index.html

```diff
@@ -92,7 +92,7 @@ <h1>I'm Adeyemi Glad to see you!</h1>
                 <a class="Linkedin-Icon" href="#Linkined.com/McEmarkees" target="_blank" ><img src="./images/Linkedin icon.png" alt="Linkedin"></a>
                 <a class="McList" href="#Mclist.com/McEmarkees" target="_blank" ><img src="./images/McList.png" alt="Linkedin"></a>
                 <a class="gitHub" href="#github.com/Emarkees" target="_blank" ><img src="./images/hangama.png" alt="GitHub"></a>
-                <a class="handwe" href="#github.com/Emarkees" target="_blank" ><img src="./images/handwe.png" alt="GitHub"></a>
+                <a class="handwe" href="#angelist.com/Emarkees" target="_blank" ><img src="./images/handwe.png" alt="GitHub"></a>
```

Comment:
- [ ] It is more semantically correct to use, in this case, this structure for your icons:

```
<ul>
  <li>
    <a>your icon</a>
  </li>
  <li>
    <a>your icon</a>
  </li>
  <li>
    <a>your icon</a>
  </li>
  ...
</ul>
```
Kindly make this improvement 👍